Makefile

```diff
@@ -25,10 +25,10 @@ _build:
 
 build: prereqs _build
 
-test: prereqs test_fmt
+test: test_fmt
 	go test -v $(GOFILES)
 
-test_fmt: prereqs
```

Comment:
By accident, it doesn't work in my WSL setup. I removed that temporarily to run make commands. 
I wasn't meant to check it in. Will revert.